ocs_ci/deployment/deployment.py

```diff
@@ -76,6 +86,40 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
         self.ocp_deployment.deploy_prereq()
         self.ocp_deployment.deploy(log_cli_level)
 
+    def deploy_ocs_via_operator(self):
+        """
+        Method for deploy OCS via OCS operator
+        """
+        logger.info("Deployment of OCS via OCS operator")
+        run_cmd(f"oc create -f {constants.OPERATOR_OLM_MANIFEST}")
+        csv = CSV(
+            name=f"ocs-operator.{self.ocs_operator_version}", kind="csv",
+            namespace=self.namespace
+        )
+        csv.wait_for_phase("Succeeded")
+        cluster_data = templating.load_yaml_to_dict(
+            self.ocs_operator_storage_cluster_cr,
+
+        )
+        cluster_data['metadata']['name'] = config.ENV_DATA[
```

Comment:
I would like to fail here if storage_cluster_name is not defined in our configuration, it will be mandatory parameter. If I will use get then it will be set to None which I don't want here.